deepchecks/utils/strings.py

```diff
@@ -253,11 +256,15 @@ def string_baseform(string: Hashable) -> Hashable:
     Returns
     -------
     str
-        string without special characters
+        original input if condition is not met or lower form alphanumeric characters of input.
     """
     if not isinstance(string, str):
         return string
-    return string.translate(DEL_MAP).lower()
+    lower_alphanumeric_form = string.translate(DEL_MAP).lower()
+    if len(lower_alphanumeric_form) > 0:
```

Comment:
maybe we want this to be optional? does that fit in every check we use this?

Reply:
Will add a boolean